.github/workflows/release-python-grpc-web.yml

```diff
@@ -0,0 +1,156 @@
+          docker-options: -e CI
+      - name: Install wheel
+        run: |
+          pip install qcs-sdk-python --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels-grpc-web
+          path: dist
+
+  linux:
+    runs-on: ubuntu-latest
+    needs: [is-python-release, should-publish-wheels]
+    env:
+      CXXFLAGS: "-std=c++11"
+    strategy:
+      matrix:
+        target: [x86_64, aarch64, ppc64le]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.12
+        architecture: x64
+    - name: Install protoc
+      uses: arduino/setup-protoc@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        version: '3.20.1'
+    - name: Patch package metadata for grpc-web
+      run: |
+        pip install toml
+        python crates/python/scripts/patch_grpc_web.py
+        cargo update
+    - name: Build wheels
+      uses: messense/maturin-action@v1
+      with:
+        target: ${{ matrix.target }}
+        manylinux: 2_28
+        args: -i ${{ matrix.python-version }} --release --manifest-path crates/python/Cargo.toml --features grpc-web --out dist -i 3.8 3.9 3.10 3.11 3.12 
+    - name: Upload wheels
+      uses: actions/upload-artifact@v3
+      with:
+        name: wheels-grpc-web
+        path: dist
+  sdist:
+    runs-on: ubuntu-latest
+    needs: is-python-release
+    env:
+      CXXFLAGS: "-std=c++11"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        default: true
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.12'
+    - name: Install protoc
+      uses: arduino/setup-protoc@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        version: '3.20.1'
+    - name: Patch package metadata for grpc-web
+      run: |
+        pip install toml
+        python crates/python/scripts/patch_grpc_web.py
+        cargo update
+    - name: Build sdist
+      uses: messense/maturin-action@v1
+      with:
+        command: sdist
+        args: --manifest-path crates/python/Cargo.toml --out dist
+    - name: Upload sdist
+      uses: actions/upload-artifact@v3
+      with:
+        name: wheels-grpc-web
+        path: dist
+
+  publish:
+    runs-on: ubuntu-latest
+    # `needs` forces this job to wait until all specified jobs
+    # are finished to run. Typically, those jobs would all have
+    # to be successful, but when combined with `if: always()`,
+    # this job is allowed to run after all the needed jobs
+    # finish, regardless of their outcome. In this case, we
+    # still make sure that at least a source distribution
+    # can be published.
+    needs: [ linux, sdist ]
+    # needs: [ macos, linux, sdist ]
```

Comment:
Does this need to depend on macos again? The latest macos build is taking over 1 hour 😞